client/connection-string/index.ts

```diff
@@ -0,0 +1,120 @@
+import * as url from 'url';
```

Comment:
Should we add .mjs/.cjs files here since you're using default export?

Reply:
Yes.